pkg/assets/assets.go

```diff
@@ -0,0 +1,67 @@
+package assets
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/bootkube/pkg/assets/internal"
+)
+
+const (
+	AssetPathCAKey           = "ssl/ca.key"
```

Comment:
I wanted to change this because I always disliked calling everything `-key.pem`. fwiw `.key` `.crt` are more universal -- but I could be convinced to switch back...